src/extlibs/hrvo/visualizer/BUILD

```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@python_tools_deps//:requirements.bzl", "requirement")
+
+# py_binary(
+#     name = "hrvo_visualizer",
+#     srcs = ["hrvo_visualizer.py"],
+#     deps = [
+#         requirement("matplotlib"),
+#         requirement("pandas"),
+#     ],
+# )
```

Comment:
nit: commented out code?

Reply:
Yeah, I was having problems getting it to work with the python visualizer. And since its not a big deal right now, and to focus on the actual integration we decided to make it a todo in the [project](https://github.com/orgs/UBC-Thunderbots/projects/11/views/1). If you think its important, we could try to spend more time and try to get the dependencies to work.

Reply:
thats all good, if its going to be immediately handled

Reply:
can you tag an TODO with an issue here?